### PR TITLE
i18n(es): update `manual-setup` & `configuration`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- What does this PR change? Give us a brief description.
Updates `manual-setup` & `configuration`

### CODE REVIEW

File: docs/src/content/docs/es/manual-setup.mdx [hunk 1]
```diff
@@ -125,8 +125,8 @@ En el futuro, planeamos soportar mejor este caso de uso para evitar la necesidad
 
 ### Usa Starlight con SSR
 
-Por el momento, Starlight no soporta el [despliegue SSR](https://docs.astro.build/es/guides/server-side-rendering/) usando los adaptadores de servidor de Astro. Esperamos poder soportar esto pronto.
+Para habilitar el SSR, sigue la guía [“Adaptadores de Renderizado bajo Demanda”](https://docs.astro.build/es/guides/server-side-rendering/) en la documentación de Astro para agregar un adaptador de servidor a tu proyecto Starlight.
 
 Puedes usar Starlight junto a páginas personalizadas renderizadas bajo demanda en tu proyecto siguiendo la guía [“Adaptadores de Renderizado bajo Demanda”](https://docs.astro.build/es/guides/server-side-rendering/) en la documentación de Astro.
```
Comment (delucis) [inline docs/src/content/docs/es/manual-setup.mdx:130]: Looks like we somehow duplicated these two paragraphs
| English | Spanish |
|---|---|
| image | image |

I guess we want to delete the second one:

```suggestion
Para habilitar el SSR, sigue la guía [“Adaptadores de Renderizado bajo Demanda”](https://docs.astro.build/es/guides/server-side-rendering/) en la documentación de Astro para agregar un adaptador de servidor a tu proyecto Starlight.
```

File: docs/src/content/docs/es/manual-setup.mdx [hunk 2]
```diff
@@ -125,8 +125,8 @@ En el futuro, planeamos soportar mejor este caso de uso para evitar la necesidad
...
 
-Por el momento, las páginas de documentación generadas por Starlight siempre son prerenderizadas, independientemente del modo de salida de tu proyecto. Esperamos poder soportar el renderizado bajo demanda para las páginas de Starlight pronto.
+Las páginas de documentación generadas por Starlight se pre-renderizan de forma predeterminada independientemente del modo de salida de tu proyecto. Para excluir tus páginas de Starlight de la pre-renderización, establece la opción de configuración [`prerender`](/es/reference/configuration/#prerender) en `false`.
```
Comment (delucis) [inline docs/src/content/docs/es/manual-setup.mdx:132]: Tiny detail but in English the link text includes “config option”:
```suggestion
Las páginas de documentación generadas por Starlight se pre-renderizan de forma predeterminada independientemente del modo de salida de tu proyecto. Para excluir tus páginas de Starlight de la pre-renderización, establece la [opción de configuración `prerender`](/es/reference/configuration/#prerender) en `false`.
```

**Review by delucis — CHANGES_REQUESTED** [review]
Thank you @dreyfus92! Noticed a couple of small details in this one.

**Comment by dreyfus92** [comment]
Thanks again Chris 🫡

**Review by delucis — APPROVED** [review]
Thanks for fixing those @dreyfus92 🙌

**Pull request merged**